back/sssscs/src/main/java/com/ib/certificate/CertificateController.java

```diff
@@ -86,12 +82,17 @@ public ResponseEntity<String> acceptRequest(@PathVariable Long id) {
 	public ResponseEntity<?> rejectRequest(@PathVariable Long id, @RequestBody String reason) {
 		CertificateRequest req = certificateRequestService.findByIdAndStatus(id, Status.PENDING);
 		
-		User issuee = userService.findById(1L); // TODO: Fetch user ID from token.
-		if (issuee == null) {
+//		System.err.println(SecurityContextHolder.getContext());
+//		System.err.println(SecurityContextHolder.getContext().getAuthentication());
+//		final User issuer = (User)(SecurityContextHolder.getContext().getAuthentication().getPrincipal());
+//		System.err.println(issuer);
+		
+		User issuer = userService.findById(1L); // TODO: Fetch user ID from token.
+		if (issuer == null) {
 			return new ResponseEntity<>("User does not exist", HttpStatus.NOT_FOUND);
 		}
 		
-		List<CertificateRequest> requests = certificateRequestService.findByIssuee(issuee);
+		List<CertificateRequest> requests = certificateRequestService.findByIssuee(issuer);
 		if (!requests.contains(req)) {
 			// return new ResponseEntity<>("Request not found", HttpStatus.NOT_FOUND);
 		}
```

Comment:
Same two comments as above in code ^^

Reply:
Altered in [e0e6910](https://github.com/magley/sssscs/pull/5/commits/e0e691099ce41198adfd10b1fb405b75e63cf07b), same as in the comments above.